src/methods/validators.rs

```diff
@@ -1,3 +1,64 @@
+//! Queries active validators on the network.
+//!
+//! Returns details and the state of validation on the blockchain.
+//!
+//! ## Examples
+//!
+//! - Get the validators for a specified epoch.
+//!
+//!     ```
+//!     use near_jsonrpc_client::{methods, JsonRpcClient};
+//!     use near_primitives::types::{EpochReference, EpochId, BlockReference, Finality};
```

Comment:
This looks a bit jarring, let's standardize indentation with lists.

Here's an example: https://github.com/near/near-jsonrpc-client-rs/blob/0a62e201d6e18dc13b6e3ff93ffead93e4d2e3e3/src/methods/block.rs#L11-L20